MekHQ/src/mekhq/campaign/Quartermaster.java

```diff
@@ -470,13 +470,13 @@ public boolean buyUnit(Entity en, int days) {
             Money cost = new Unit(en, getCampaign()).getBuyCost();
             if (getCampaign().getFinances().debit(TransactionType.UNIT_PURCHASE, getCampaign().getLocalDate(),
                     cost, "Purchased " + en.getShortName())) {
-                getCampaign().addNewUnit(en, false, days);
+                getCampaign().addNewUnit(en, false, days, 3);
```

Comment:
It might make sense to retain the old version like this. To avoid having to update all the old calls.
```
public Unit addNewUnit(Entity en, boolean allowNewPilots, int days) {
   return addNewUnit(en, allowNewPilots, days, 3);
}
```

Reply:
Oh, that is such a good idea!